modular_pariah/modules/skrell/code/modules/mob/living/carbon/human/species_types/skrell.dm

```diff
@@ -0,0 +1,179 @@
+	mutanteyes = /obj/item/organ/eyes/skrell
+	mutantlungs = /obj/item/organ/lungs/skrell
+	mutantheart = /obj/item/organ/heart/skrell
+	mutantliver = /obj/item/organ/liver/skrell
+	mutanttongue = /obj/item/organ/tongue/skrell
+
+	bodypart_overrides = list(
+		BODY_ZONE_HEAD = /obj/item/bodypart/head/skrell,
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest/skrell,
+		BODY_ZONE_L_ARM = /obj/item/bodypart/l_arm/skrell,
+		BODY_ZONE_R_ARM = /obj/item/bodypart/r_arm/skrell,
+		BODY_ZONE_L_LEG = /obj/item/bodypart/l_leg/skrell,
+		BODY_ZONE_R_LEG = /obj/item/bodypart/r_leg/skrell,
+	)
+
+/datum/species/skrell/spec_life(mob/living/carbon/human/skrell_mob, delta_time, times_fired)
+	. = ..()
+	if(skrell_mob.nutrition > NUTRITION_LEVEL_ALMOST_FULL)
+		skrell_mob.set_nutrition(NUTRITION_LEVEL_ALMOST_FULL)
+
+/datum/species/skrell/prepare_human_for_preview(mob/living/carbon/human/human)
+	human.dna.features["mcolor"] = sanitize_hexcolor(COLOR_BLUE_GRAY)
+	human.update_body()
+	human.update_body_parts()
+
+// Preset Skrell species
+/mob/living/carbon/human/species/skrell
+	race = /datum/species/skrell
+
+// Skrell bloodbag, for S type blood
+/obj/item/reagent_containers/blood/skrell
+	blood_type = "S"
+
+// Copper restores blood for Skrell instead of iron.
+/datum/reagent/copper/on_mob_life(mob/living/carbon/C, delta_time)
+	if((isskrell(C)) && (C.blood_volume < BLOOD_VOLUME_NORMAL))
+		C.blood_volume += 0.5 * delta_time
+	..()
+
+// Organ for Skrell head tentacles.
+/obj/item/organ/external/headtails
+	zone = BODY_ZONE_HEAD
+	slot = ORGAN_SLOT_EXTERNAL_HEADTAILS
+	layers = EXTERNAL_FRONT | EXTERNAL_ADJACENT
+	dna_block = DNA_HEADTAILS_BLOCK
+	feature_key = "headtails"
+	preference = "feature_headtails"
+
+/obj/item/organ/external/headtails/can_draw_on_bodypart(mob/living/carbon/human/human)
+	. = TRUE
+	if(istype(human.head) && (human.head.flags_inv & HIDEHAIR))
+		. = FALSE
+	if(istype(human.wear_mask) && (human.wear_mask.flags_inv & HIDEHAIR))
+		. = FALSE
+	var/obj/item/bodypart/head/our_head = human.get_bodypart(BODY_ZONE_HEAD)
+	if(our_head && !IS_ORGANIC_LIMB(our_head))
+		. = FALSE
+
+/obj/item/organ/external/headtails/get_global_feature_list()
+	return GLOB.headtails_list
+
+
+/obj/item/organ/heart/skrell
+	name = "skrellian heart"
+	icon = 'modular_pariah/modules/skrell/icons/obj/skrell_organ.dmi'
+	icon_state = "heart"
+
+/obj/item/organ/brain/skrell
+	name = "spongy brain"
+	icon = 'modular_pariah/modules/skrell/icons/obj/skrell_organ.dmi'
+	icon_state = "brain2"
+
+/obj/item/organ/eyes/skrell
+	name = "amphibian eyes"
+	desc = "Large black orbs."
+	icon = 'modular_pariah/modules/skrell/icons/obj/skrell_organ.dmi'
+	icon_state = "eyes"
+	flash_protect = FLASH_PROTECTION_SENSITIVE
+
+/obj/item/organ/lungs/skrell
+	name = "skrell lungs"
+	icon = 'modular_pariah/modules/skrell/icons/obj/skrell_organ.dmi'
+	icon_state = "lungs"
+	safe_plasma_max = 40
+	safe_co2_max = 40
+
+	cold_message = "You can't stand the freezing cold with every breath you take!"
```

Comment:
This and `hot_message` need a bit of tweaking:
![image](https://user-images.githubusercontent.com/57483089/164076961-6b7d9af5-2c43-434a-8bd0-a4747adfaf3d.png)

Reply:
Wasnt necessary afterall